html/ImdbHtml.html

```diff
@@ -0,0 +1,69 @@
+
+<html>
+	
+	<head>
+	
+	<link type="text/css" rel="stylesheet" href="‪C:\Users\Gilron\Desktop\ImdbWebApplication\CSS\ImdbCSS.css" MEDIA="screen" />

```

Comment:
@oregilron Use [relative path](http://www.coffeecup.com/help/articles/absolute-vs-relative-pathslinks/) pay attention to the ```../``` prefix.